<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWidget</class>
 <widget class="QWidget" name="TestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>编辑文章</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="toolButton_9">
       <property name="text">
        <string>字体刷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_10">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_11">
       <property name="text">
        <string>颜色刷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_12">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_13">
       <property name="text">
        <string>插如图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>颜色编辑器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>字体编辑器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>查找&amp;F</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_2</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_14">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="toolButton_3">
       <property name="text">
        <string>左对齐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>居中</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_4">
       <property name="text">
        <string>右对齐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_5">
       <property name="text">
        <string>下划线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>加粗</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_6">
       <property name="text">
        <string>放大</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_7">
       <property name="text">
        <string>变小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_8">
       <property name="text">
        <string>红字</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>插入网页</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>简介</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>预览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>toolButton_6</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>toolButton_10</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>toolButton_5</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>toolButton_4</tabstop>
  <tabstop>toolButton_13</tabstop>
  <tabstop>toolButton_9</tabstop>
  <tabstop>toolButton_7</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>toolButton_11</tabstop>
  <tabstop>toolButton_12</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>toolButton_8</tabstop>
  <tabstop>toolButton_14</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
